internal/rbd/driver/driver.go

```diff
@@ -159,6 +160,10 @@ func (r *Driver) Run(conf *util.Config) {
 
 	if conf.IsControllerServer {
 		r.cs = NewControllerServer(r.cd)
+		r.cs.ClusterName = conf.ClusterName
+		if conf.ClusterName != "" {
+			log.DefaultLog("%q is set to %q", k8s.ClusterNameKey, conf.ClusterName)
```

Comment:
I do not think it is very useful to log this. The message will be a little confusing, because the key is not set here at all, just an attribute of the controller-server